pwndbg/heap/ptmalloc.py

```diff
@@ -1398,7 +1402,14 @@ def main_arena(self):
                     for addr in range(start, end, pwndbg.gdblib.arch.ptrsize):
                         found = False
                         tmp_arena = self.malloc_state(addr)
-                        tmp_next = int(tmp_arena["next"])
+                        try:
+                            tmp_next = int(tmp_arena["next"])
+                        except Exception:
```

Comment:
is this gdb.MemoryError or gdb.error that happens here? Can we be more explicit with the exceptions that we want to catch? 🤐